assets/js/game.js

```diff
@@ -0,0 +1,20 @@
+function Game() {
+  this.heroes = [];
+  this.enemies = [];
```

Comment:
To get started why not just try `this.enemies = [enemies[0]];`?

That would allow you to test your templates and get moving. Then when the time comes, you have the game set a random enemy instead of the same one all the time.